base/trace_event/common

```diff
@@ -0,0 +1 @@
+Subproject commit e40c41030f44cbd5b6f54081436620f43c3bb08a
```

Comment:
eh, it's not necessary, I have no idea why it's here, I will remove it and resubmit.